releasePipeline/release.md

```diff
@@ -26,6 +26,90 @@ For each of the Kubernetes Tekton command, you can follow with tkn -n galasa-bui
 
 5. Run [20-build-all-code..sh](./20-build-all-code.sh). When prompted, choose the '`release`' option.
 6. Run [28-run-regression-tests.sh](./28-run-regression-tests.sh). 
+All the tests must pass before moving on.
+For the ones which fail, run them individually (ask someone how to do this). As currently some tests pass if run a second time due to the vaguaries of system resource availability. Also make sure @hobbit1983's VM image isn't down.
 
```

Comment:
instead of saying ask someone how should we not say what was in the current release process? i.e:
```
3. If there are any failures from the regression testing - Amend 29-regression-reruns.yaml and pipelines/regression-reruns.yaml. Add the tests that failed, to run them again.
4. Run `kubectl -n galasa-build create -f 29-regression-reruns.yaml` - Retest the failing tests.
5. Repeat as required.
```

Reply:
... I copied all the previous READMe of the release process into this document, beyond the bits you did !
But sure. I can add this...